docs/software/cloud.md

```diff
@@ -230,9 +230,38 @@ Nextcloud gives you access to all your files wherever you are. Store your docume
 
     Your user data directory will stay after deinstallation. As well a database backup will be saved to your user data directory. Thus you can easily restore your instance by reinstalling Nextcloud and restore the database dump.
 
+    **How can I check my OPcache status?**
+
+    PHP uses a so called *OPcache* to store PHP scripts in optimised *opcode* format in RAM, which speeds up browser access as the raw PHP scripts do not need to be read from disk and parsed first.  
+    The Nextcloud admin panel includes a check for sufficient OPcache settings and in case shows a recommendation for settings to apply. If such appears, we recommend to apply them with an own configuration file, e.g.:
+
+    ```sh
+    echo -e '; Custom Nextcloud OPcache settings\n; priority=99\nopcache.memory_consumption=128' > /etc/php/7.4/mods-available/custom.ini
+    phpenmod custom
+    ```
```

Comment:
I rephrased it so that it will still be correct with NC24 and in case the NC23 backport or the changed checks.